### ISSUE TITLE
Improve usability of seek dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cplussharp/graph-studio-next#96]
```
I'd like to have this feature in the programm:

Dialog shouldn't close on every seek as it's modeless.
Tab order and keyboard accelerators should be added.

The feature is important for me because:
Makes it easier to do multiple seeks easily and access dialog from the keyboard.

The feature is important for other users because:


```

Original issue reported on code.google.com by `mike.cop...@gmail.com` on 15 Jan 2013 at 5:15
